src/main/kotlin/no/nav/syfo/client/person/kontaktinfo/KontaktinformasjonClient.kt

```diff
@@ -62,6 +68,10 @@ class KontaktinformasjonClient(
     companion object {
         const val PERSON_PATH = "/syfoperson/api/person"
         const val PERSON_KONTAKTINFORMASJON_PATH = "$PERSON_PATH/kontaktinformasjon"
+
+        const val CACHE_KONTAKTINFORMASJON_KEY_PREFIX = "person-kontaktinformasjon-"
+        const val CACHE_KONTAKTINFORMASJON_EXPIRE_SECONDS = 600
```

Comment:
Skal det være `3600` her? Hvis ikke, hvorfor så kort?

Reply:
Diskuterte dette noe med @janzh og han mente den kanskje burde ligge mellom 5-15 minutter siden den endres oftere enn adressebeskyttelse (eller at det er mer kritisk om cachen er utdatert). Tenker du den burde være lik som adressebeskyttelse?

Reply:
Jeg tenkte den kunne være lik som adressebeskyttelse, men hvis dere allerede har diskutert det, er det bra nok for meg 👍